go/service/account_test.go

```diff
@@ -74,3 +74,28 @@ func TestRecoverUsernameWithEmail(t *testing.T) {
 	})
 	require.NoError(t, err)
 }
+
+func TestContactSettingsAPI(t *testing.T) {
+	tc := libkb.SetupTest(t, "cset", 3)
+	defer tc.Cleanup()
+
+	_, err := kbtest.CreateAndSignupFakeUser("cset", tc.G)
+	require.NoError(t, err)
+
+	handler := NewAccountHandler(nil, tc.G)
+	ctx := context.Background()
+	res, err := handler.UserGetContactSettings(ctx)
+	require.NoError(t, err)
+
+	err = handler.UserSetContactSettings(ctx, keybase1.ContactSettings{
+		Enabled:              true,
+		AllowFolloweeDegrees: 2,
+	})
```

Comment:
better to also test setting Teams JSON here. i didn't find a easy way to create a team in a test